modules/Sources/Features/Settings/Views/About.swift

```diff
@@ -18,13 +18,35 @@ public struct About: View {
     
     public var body: some View {
         WithViewStore(store) { viewStore in
-            VStack {
-                Text(L10n.Settings.version(viewStore.appVersion, viewStore.appBuild))
-                    .foregroundColor(Asset.Colors.Mfp.fontDark.color)
+            VStack(alignment: .leading, spacing: 45) {
+                HStack(spacing: 15) {
+                    Asset.Assets.zashiLogo.image
+                        .resizable()
+                        .frame(width: 62, height: 81)
+                    
+                    Asset.Assets.zashiLogoText.image
+                        .resizable()
+                        .frame(width: 110, height: 30)
+                }
+                VStack(alignment: .leading, spacing: 25) {
+                    Text(L10n.Settings.version(viewStore.appVersion, viewStore.appBuild))
+                        .font(Font.system(size: 14, weight: .bold))
+                        .foregroundColor(Asset.Colors.Mfp.fontDark.color)
+                    Text("Send and receive ZEC on Zashi! Zashi is a minimal-design, self-custody, ZEC-only shielded wallet that keeps your transaction history and wallet balance private. Built by Zcashers, for Zcashers. Developed and maintained by Electric Coin Co., the inventor of Zcash, Zashi features a built-in user-feedback mechanism to enable more features, more quickly.")
+                        .foregroundColor(Asset.Colors.Mfp.fontDark.color)
+                        .font(Font.system(size: 14, weight: .regular))
```

Comment:
There are Inter fonts in most designs, please replace the system one.

Reply:
![Simulator Screenshot - iPhone 14 - 2023-09-07 at 18 02 02](https://github.com/zcash/secant-ios-wallet/assets/144026640/65ea6d04-dccd-4839-bbbc-5006faf46c01)

Updated About Screen UI